apps/src/code-studio/pd/application/teacher/AdditionalDemographicInformation.jsx

```diff
@@ -45,8 +45,8 @@ const AdditionalDemographicInformation = props => {
               {data.csaAlreadyKnow === 'No' && (
                 <p style={styles.error}>
                   We don’t recommend this program for teachers completely new to
-                  CS. If possible, consider teaching CS Principles in the
-                  upcoming school year and applying for our CS Principles
+                  computer science. If possible, consider teaching CS Principles
+                  in the upcoming school year and applying for our CS Principles
                   Professional Learning program. If this is not possible, plan
                   to spend at least 40 hours learning foundational CS concepts
                   prior to attending our professional learning for CSA.
```

Comment:
In the spec, the wording looks a bit different: "We don’t recommend this program for teachers completely new to computer science. Consider starting with CS Principles Professional Learning or plan for additional onboarding in preparation for this program." Were there changes since then?